### DISCUSSION TITLE
Binding to ExpandoObject

### DISCUSSION THREAD

**Discussion by YSW33** [AvaloniaUI/Avalonia#12518] [Q&A]
Do I have this possibility to use ExpandoObject as a Source?
What I test didn't work.
WPF testing feasible.

![image](https://github.com/AvaloniaUI/Avalonia/assets/141573398/7992e32f-7e09-4717-80e2-45d05067b10d)
![image](https://github.com/AvaloniaUI/Avalonia/assets/141573398/66817317-f0b3-427b-889a-b1ce43a2dc23)
![image](https://github.com/AvaloniaUI/Avalonia/assets/141573398/9cb7284e-75f9-4577-a63c-138df9998083)
![image](https://github.com/AvaloniaUI/Avalonia/assets/141573398/e62f3b38-9d91-4e18-98c0-8563ecee65d0)

**Comment by timunie** [comment]
probably you can write a behavior to to what you need to do. Check out samples: https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors

**Reply by YSW33** [reply]
Thank you，
I'll give it a try。

**Comment by wosiyourfather** [comment]
兄弟在哪里工作啊 最近也在搞这个跨平台的啊  加个微信一起交流交流啊 gy491911214
